generated/typespec-ts/src/parameters.ts

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { RequestParameters } from "@azure-rest/core-client";
+import {
+  CreateChatCompletionRequest,
+  CreateFineTuningJobRequest,
+  CreateCompletionRequest,
+  CreateEditRequest,
+  CreateEmbeddingRequest,
+  CreateFineTuneRequest,
+  CreateImageRequest,
+  CreateModerationRequest,
+} from "./models";
+
+export interface TranscriptionsCreateBodyParam {
+  body: TranscriptionsCreateFormBody;
+}
+
+export interface TranscriptionsCreateFormBody {
+  file: string;
+  model: string | "whisper-1";
+  prompt?: string;
+  response_format?: "json" | "text" | "srt" | "verbose_json" | "vtt";
+  temperature?: number;
+  language?: string;
+}
+
+export interface TranscriptionsCreateMediaTypesParam {
+  contentType: "multipart/form-data";
```

Comment:
Though we can generate multipart/form-data, but I'm not sure if this is really working.